docs/dataset/advanced-usage/dataset-formatting/computer-vision.md

```diff
@@ -213,7 +246,7 @@ Kolena supports `mov`, `mp4`, `mpeg` and other web browser supported video types
     Bounding box visualization only works for videos with `5`, `15`, `29.97` and `59.94` frame rates.
```

Comment:
I should've done this already but we do support other frame rates provided that users upload a `frame_rate` col after [this marina change](https://github.com/kolenaIO/marina/pull/3953). Created [a ticket](https://linear.app/kolena/issue/KOL-7448/documentation-on-frame-rate-support) that I'll do this cycle.